pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -524,17 +546,19 @@ func Run(ps *pubsub.PubSub) {
 
 	switch flag.Args()[0] {
 	case "setupVaults":
-		if err = setupFscryptEnv(); err != nil {
-			log.Fatal("Error in setting up fscrypt environment:", err)
-		}
-		if err = setupVault(defaultImgVault); err != nil {
-			log.Fatalf("Error in setting up vault %s:%v", defaultImgVault, err)
-		}
-		if err = setupVault(defaultCfgVault); err != nil {
-			log.Fatalf("Error in setting up vault %s %v", defaultCfgVault, err)
+		//start with an assumption that nothing needs to be done
+		persistFsType := ""
+		pBytes, err := ioutil.ReadFile(evePersistTypeFile)
+		if err == nil {
+			persistFsType = string(pBytes)
```

Comment:
Need to strip the trailing CR

Reply:
> Need to strip the trailing CR

I modified the echo command in storage-init.sh to pass "-ne" flags, so this should be fine here. But yetus is complaining of "-ne" in POSIX echo command. To me that looks more reasonable, since comparison (which may be in multiple places) need not bother about stripping whitespaces.

Reply:
Hey @cshari-zededa -- I think a bit of defensive programming on the Go side would be useful anyway. At any rate -- I committed the change to storage-init that is now doing the right thing there -- but I didn't update echo with these additional arguments.

If you feel really strongly about it -- I'm fine either way.